static/app/types/hooks.tsx

```diff
@@ -170,7 +170,6 @@ export type ComponentHooks = {
   'component:disabled-member': () => React.ComponentType<DisabledMemberViewProps>;
   'component:disabled-member-tooltip': () => React.ComponentType<DisabledMemberTooltipProps>;
   'component:enhanced-org-stats': () => React.ComponentType<OrganizationStatsProps>;
-  'component:escalating-issues-banner-feedback': () => React.ComponentType<QualitativeIssueFeedbackProps>;
```

Comment:
Do we still want this? I'm looking at the getsentry usages of the flag too and am not sure if I should remove the feedback banner.

Reply:
For posterity, I asked and Brian said to keep it for now.